project/src/main/java/com/google/ensemble/servlets/UploadServlet.java

```diff
@@ -50,7 +50,7 @@
 
   private static final String INPUT_NAME_PROJECT = "project";
   private static final String INPUT_NAME_FILENAME = "filename";
-  private static final String INPUT_NAME_FILES = "files";
+  private static final String INPUT_NAME_FILES = "file";
```

Comment:
Should this be `INPUT_NAME_FILE` now?

Reply:
Fixed, added possible new issue for frontend to support multiple simultaneous uploads